website/views.py

```diff
@@ -3993,17 +3993,23 @@ def trademark_detailview(request, slug):
 #         mail.logout()
 def update_bch_address(request):
     if request.method == "POST":
+        selected_crypto = request.POST.get("selected_crypto")
         new_address = request.POST.get("new_address")
-        if new_address:
+        if selected_crypto and new_address:
             try:
                 user_profile = request.user.userprofile
-                user_profile.crypto_address = new_address
+                if selected_crypto == "Bitcoin":
+                    user_profile.btc_address = new_address
+                elif selected_crypto == "Ethereum":
+                    user_profile.eth_address = new_address
+                elif selected_crypto == "BitcoinCash":
+                    user_profile.bch_address = new_address
```

Comment:
Optional: it looks like a candidate for `match` statement (https://peps.python.org/pep-0636/#appendix-a-quick-intro)

Reply:
Okay, is match statement better than if else ?

Reply:
It may be more readable, I'm leaving it to you to decide whether it's better or not.

Reply:
![image](https://github.com/OWASP-BLT/BLT/assets/108613416/4405bae0-047e-4441-b8ca-54eb82f78383)
I made the changes , It is better if else statement had repetition